rcsbapi/dev_tools/update_schema.py

```diff
@@ -0,0 +1,144 @@
+"""Update the distribution json files; for developer use only
+This is currently not in use. """
```

Comment:
Can you put usage instructions up here?

Reply:
Totally forgot to do that, I added them in!